submission/persistence-mongo/src/main/kotlin/ac/uk/ebi/biostd/persistence/doc/model/DocSubmissionDraft.kt

```diff
@@ -25,6 +25,7 @@ data class DocSubmissionDraft(
 
     enum class DraftStatus {
         ACTIVE,
-        PROCESSING
+        PROCESSING,
+        DELETED
```

Comment:
Rather than deleted, I think we should call this "PROCESSED" or "ACCEPTED" since I think it reflects better what really happened with the draft: it was submitted and the system accepted it. Also, I think it's counterintuitive that we have something marked as deleted instead of actually deleting it.

Reply:
It depends how do you see the business case and in this case a draft,  IMO it is a peace of data, like record that user can created. It can be active , block or deleted.  IMO We do not even process draft of accept them as it  like a memory that can have sometime, can be block for changes or can be deleted. 

IMO the concept of accepted, processing or submitting is related to a submission request not to a DRAFT.